include/xsimd/arch/xsimd_neon.hpp

```diff
@@ -2053,7 +2053,7 @@ namespace xsimd
         template <class A, class T>
         inline batch<T, A> bitwise_lshift(batch<T, A> const& lhs, int n, requires_arch<neon>) noexcept
         {
-            constexpr std::size_t size = sizeof(typename batch<T, A>::value_type) * 8;
+            constexpr int size = sizeof(typename batch<T, A>::value_type) * 8;
             assert(0 <= n && n < size && "index in bounds");
             return detail::bitwise_lshift_impl(lhs, n, ::xsimd::detail::make_int_sequence<size>());
```

Comment:
Warning comparison of signed `n` and unsigned `size`.
Warning make_int_sequence<`size`> is unsigned, but the bitwise_lshift_impl expects `int`.